<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TownEventDialog</class>
 <widget class="QDialog" name="TownEventDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Town event</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_16">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>511</width>
         <height>351</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
        <item>
         <widget class="QLineEdit" name="eventNameText">
          <property name="placeholderText">
           <string>Event name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="eventMessageText">
          <property name="placeholderText">
           <string>Type event message text</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>511</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="eventFirstOccurrenceText">
            <property name="text">
             <string>Day of first occurrence</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="eventFirstOccurrence"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="eventRepeatAfterText">
            <property name="text">
             <string>Repeat after (0 = no repeat)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="eventRepeatAfter"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>529</x>
         <y>9</y>
         <width>141</width>
         <height>421</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="playersAffectedText">
          <property name="text">
           <string>Affected players</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="playersAffected">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="eventAffectsHuman">
          <property name="text">
           <string>affects human</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="eventAffectsCpu">
            <property name="text">
             <string>affects AI</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="resourcesTab">
      <attribute name="title">
       <string>Resources</string>
      </attribute>
      <widget class="QTableWidget" name="resourcesTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>661</width>
         <height>421</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column/>
       <column/>
      </widget>
     </widget>
     <widget class="QWidget" name="buildingsTab">
      <attribute name="title">
       <string>Buildings</string>
      </attribute>
      <widget class="QTreeView" name="buildingsTree">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>661</width>
         <height>421</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QWidget" name="creaturesTab">
      <attribute name="title">
       <string>Creatures</string>
      </attribute>
      <widget class="QTableWidget" name="creaturesTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>661</width>
         <height>421</height>
        </rect>
       </property>
       <property name="rowCount">
        <number>7</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
